apps/email/js/sync.js

```diff
@@ -7,7 +7,7 @@ define(function(require) {
       appSelf = require('app_self'),
       evt = require('evt'),
       mozL10n = require('l10n!'),
-      notificationHelper = require('shared/js/notification_helper');
+      NotificationHelper = require('shared/js/notification_helper');
```

Comment:
Please, stick to the naming convention - variables start with lower case letter.